### DISCUSSION TITLE
How secure is use this tool?

### DISCUSSION THREAD

**Discussion by joaquin-arrazola-jala** [yuezk/GlobalProtect-openconnect#195] [Q&A] [answered]
Hello, I would like to know how secure is use this tool, regarding to the credentials used to login to the VPN, are the prompted login page goes by https?

**Comment by diomenezes** [comment]
I my case, I only use my Public IP

**Comment by yuezk** [comment, marked as answer]
@joaquin-arrazola-jala Yes, it will force to use HTTPS.

https://github.com/yuezk/GlobalProtect-openconnect/blob/2761f7521a5e8e4928fd766af28b94d763c15708/GPClient/portalauthenticator.cpp#L18-L19

https://github.com/yuezk/GlobalProtect-openconnect/blob/2761f7521a5e8e4928fd766af28b94d763c15708/GPClient/portalauthenticator.cpp#L18-L19
